src/Stratis.Bitcoin.IntegrationTests/RPC/RPCTestsMutable.cs

```diff
@@ -21,11 +21,18 @@ public void TestRpcGetBalanceIsSuccessful()
                 builder.StartAll();
                 RPCClient rpcClient = node.CreateRPCClient();
 
-                TestHelper.MineBlocks(node, 2);
-                TestHelper.WaitLoop(() => node.FullNode.GetBlockStoreTip().Height == 2);
+                int maturity = (int)KnownNetworks.RegTest.Consensus.CoinbaseMaturity;
+
+                // This shouldn't be necessary but is required to make this test pass.
+                // - looks like a bug in the wallet?
+                maturity--; 
```

Comment:
@bokobza I think this is a off by one bug in the wallet? This tests on bitcoin core with the decrement so I think we have bug or maybe them 😜

Reply:
>or maybe them

I like your optimism 😄 if this was the case then we helped bitcoin ;)